### ISSUE TITLE
We need specs for WS protocol API since we have none

### ISSUE THREAD

**Issue by yivo** [openware/peatio#705]

**Comment by yivo** [comment]
PR to continue work on #730.

**Comment by dinesh-skyach** [comment]
@yivo I tried running the Websocket API and I think the API is broken. Authentication worked perfectly but when creating/trading orders I get this error while receiving the details
```    
Error on receiving orders.
 #<TypeError: no implicit conversion of Bunny::MessageProperties into String>
```

**Comment by yivo** [comment]
@dinesh-skyach Great! Thanks!

**Comment by yivo** [comment]
Closed via #1322.
